src/main/resources/liquibase/workbench_changelog/18_4_0.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v18.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			  <sqlCheck expectedResult="0">
+                SELECT count(*) FROM workbench_tool WHERE name = 'manage_metadata'
+            </sqlCheck>
+        </preConditions>
+		<comment>
+			Add Manage Metadata sidebar item.
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO workbench_tool (`name`, `group_name`, `title`, `version`, `tool_type`, `path`, `parameter`, `user_tool`)
+			VALUES ('manage_metadata', 'manage_metadata', 'Manage Metadata', '1.0', 'WEB', '/ibpworkbench/controller/jhipster#metadata-manager', '', 0);
+
+			SET @sidebar_category_id = (SELECT sidebar_category_id FROM workbench_sidebar_category where sidebar_category_name = 'crop_administration');
+
+			INSERT INTO workbench_sidebar_category_link (`tool_name`, `sidebar_category_id`, `sidebar_link_name`, `sidebar_link_title`, `rank`)
+			VALUES ('manage_metadata', @sidebar_category_id, 'manage_metadata', 'Manage Metadata', 2);
+
+			SET @permission_id = (SELECT permission_id FROM permission where name = 'CROP_MANAGEMENT');
+			SET @workbench_sidebar_category_link_id =  (SELECT sidebar_category_link_id FROM workbench_sidebar_category_link where tool_name = 'manage_metadata');
+
+			INSERT INTO permission (`name`, `description`, `parent_id`, `workbench_sidebar_category_link_id`)
+			VALUES ('MANAGE_METADATA', 'Manage Metadata', @permission_id , @workbench_sidebar_category_link_id);
```

Comment:
should not be better to name the permission MANAGE_CROP_METADATA ? I think that we could have another one to manage program metadata in the future

Reply:
done.